Wikipedia/Code/WebViewController.m

```diff
@@ -539,11 +578,11 @@ - (void)scrollToAndFocusOnSelectedMatch {
                                                  }
                                                  newOffsetY += [self.webView iOS12yOffsetHack];
                                                  CGPoint centeredOffset = CGPointMake(self.webView.scrollView.contentOffset.x, newOffsetY);
-                                                 [self.webView.scrollView wmf_safeSetContentOffset:centeredOffset
-                                                                                          animated:YES
-                                                                                        completion:^(BOOL done) {
-                                                                                            self.disableMinimizeFindInPage = NO;
-                                                                                        }];
+                                                 [self scrollToOffset:centeredOffset
+                                                             animated:YES
+                                                           completion:^{
+                                                               self.disableMinimizeFindInPage = NO;
```

Comment:
Not sure why but there's now a find in page bug - it automatically dismisses after typing a couple of characters.

Reply:
Another good catch - looks like the issue is caused by the completions stacking up and all being called on the first animation callback. This should wait until the last completion is called before setting `self.disableMinimizeFindInPage = NO;` Will fix

Reply:
It looks like `disableMinimizeFindInPage` was there to filter out non-user scroll events from `scrollViewDidScroll` instead of using `scrollViewWillBeginDragging`. Neither of those are necessary when using the built in method on `UIScrollView`. Fixed